backend/editor.go

```diff
@@ -162,9 +162,11 @@ func getClipboard() (string, error) {
 }
 
 func (e *Editor) Init() {
+	log.Info("Initializing")
```

Comment:
There aren't that many Info-level logs though, and it's the only "Initializing" text printed afaict:
```
13:57 ~/code/go/src/github.com/limetext/lime $ ag Initializing
backend/editor.go
165: log.Info("Initializing")
13:57 ~/code/go/src/github.com/limetext/lime $```